src/registry.js

```diff
@@ -0,0 +1,187 @@
+const {reportFromAdvisory} = require('./issues/utils');
+const {loadNpmConfigs} = require('./files');
+
+const DEFAULT_REGISTRY_URL = 'https://registry.npmjs.org/';
+let registriesInfo = [];
+let fetch = () => {};
+
+const replaceEnvVars = (str) =>
+  str.replace(/\${([^}]+)}/g, (match, variableName) => process.env[variableName] || '');
```

Comment:
FYI, I am getting an error here when running `sandworm audit`:

```
❯ sandworm audit
Sandworm 🪱
Security and License Compliance Audit
⠋ Building dependency graph...
❌ Failed: str.replace is not a function
TypeError: str.replace is not a function
    at replaceEnvVars (/Users/david/.nvm/versions/node/v18.12.1/lib/node_modules/@sandworm/audit/src/registry.js:9:7)
    at /Users/david/.nvm/versions/node/v18.12.1/lib/node_modules/@sandworm/audit/src/registry.js:17:28
    at Array.forEach (<anonymous>)
    [...]
```

In my case, the error is happening because `str` is a boolean (`true`) rather than a string. (The `key` is `"color"`).